s3/replication/replicator/README.md

```diff
@@ -56,3 +56,10 @@ Run load transfer tests.
 ```sh
 python3 tests/system/load_transfer_test.py
 ```
+
+
+Note :
```

Comment:
Add it as below section

## Configurations
...note below goes here...

Reply:
done